app/controllers/trips/user_trips_controller.rb

```diff
@@ -1,5 +1,25 @@
-class Projects::ProjectUsersController < ApplicationController
+class Trips::UserTripsController < ApplicationController
+    before_action :authenticate_user!
+    before_action :set_trip
```

Comment:
może tu też only? będzie klarowniej chyba

Reply:
to to samo, co w poprzednim komenatrzu ;)